### DISCUSSION TITLE
Why my bot after restart does not see any members?

### DISCUSSION THREAD

**Discussion by GrzegorzKrug** [Rapptz/discord.py#5982] [Q&A] [answered]
I know that was update to 1.5.1
but honestly I did not upgrade discord.py and after restarting bot.
It can see only himself
`ctx.guild.members`

https://github.com/GrzegorzKrug/discord_bot/commit/de215e547364bba6761b43dafe14aa9cc56ae8f5

**Comment by nxtlo** [comment]
What do you mean by **It can see only himself** ? Are you getting errors or anything helpful to describe your issue?

**Comment by Rapptz** [comment, marked as answer]
This is a duplicate of https://github.com/Rapptz/discord.py/issues/5867

Please consider reading the [documentation on intents fully](https://discordpy.readthedocs.io/en/latest/intents.html)

**Reply by GrzegorzKrug** [reply]
Yes, that was the problem, I restarted bot and it just stoped working, I guess it was runnig so long on aws it had all in cache. I did updates, thanks

#closes5982
